js/app.js

```diff
@@ -0,0 +1,91 @@
+// sign-in input validator
+
+const mailInput = document.querySelector(".sign-up__form--input input");
+const mailButton = document.querySelector(".sign-up__form--input button");
+const inputContainer = document.querySelector(".sign-up__form--input");
+const newAlert = document.createElement("h5");
+
+const inputValidator = () => {
+  if (
+    mailInput.value.indexOf("@") !== -1 &&
+    mailInput.value.indexOf(".") !== -1 &&
+    mailInput.value.length > 6
+  ) {
+    mailInput.style.border = "1px solid green";
+  } else {
+    mailInput.style.removeProperty("border");
+  }
+
+  if (mailButton.nextElementSibling === newAlert) {
+    newAlert.parentElement.removeChild(newAlert);
```

Comment:
Unikamy takich operacji, tzn:

`.parentElement` -> `removeChild`

z tego względu, że są zależne od struktury DOM. Jeśli chcesz złapać jakiś element z poziomu JSa to złap go po `id` albo po `data-attribute`